nikola/plugins/task/tags.py

```diff
@@ -105,8 +105,7 @@ def gen_tasks(self):
             intersect = set(tags.keys()) & set(categories.keys())
             if len(intersect) > 0:
                 for slug in intersect:
-                    utils.LOGGER.error("Category '{0}' and tag '{1}' both have the same slug '{2}'!".format('/'.join(categories[slug]), tags[slug], slug))
```

Comment:
But a post cannot be in multiple categories. After looking at it again, I think the `join` should have never been there in the first place. I think you can just do `.format(categories[slug], ...`.